testsuite/MDAnalysisTests/core/test_topologyobjects.py

```diff
@@ -157,16 +164,13 @@ def test_angle(self, PSFDCD):
     def test_angle_repr(self, PSFDCD):
         angle = PSFDCD.atoms[[30, 10, 20]].angle
 
-        assert_equal(repr(angle), '<Angle between: Atom 30, Atom 10, Atom 20>')
+        assert_equal(repr(angle), "<Angle between: Atom 30, Atom 10, Atom 20>")
 
     def test_angle_180(self):
         # we edit the coordinates, so make our own universe
         u = mda.Universe(PSF, DCD)
         angle = u.atoms[210].angles[0]
-        coords = np.array([[1, 1, 1],
-                           [2, 1, 1],
-                           [3, 1, 1]],
-                          dtype=np.float32)
+        coords = np.array([[1, 1, 1], [2, 1, 1], [3, 1, 1]], dtype=np.float32)
```

Comment:
perhaps should be this:

`coords = np.array([[1, 1, 1], [2, 1, 1], [3, 1, 1],], dtype=np.float32)`

(with a trailing comma to maintain 3x3 format)